lib/warehouse/ecto_enums.ex

```diff
@@ -0,0 +1,25 @@
+import EctoEnum
+
+defenum(AreaEnum, :area, [:assembly, :receiving, :shipped, :shipping, :storage, :transit])
+
+defenum(PayableAccountType, :payable_account_type, [:net_0_payable, :net_30_payable, :net_45_payable, :net_60_payable])
```

Comment:
This can be removed now that I'm not adding all the different schemas we have in here, only those required for the read-only functionality at this time.